src/libs/actions/Policy.ts

```diff
@@ -665,7 +665,7 @@ function setWorkspaceApprovalMode(policyID: string, approver: string, approvalMo
 }
 
 function setWorkspacePayer(policyID: string, reimburserEmail: string, reimburserAccountID: number) {
-    const policy = ReportUtils.getPolicy(policyID);
+    const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] ?? ({} as Policy);
```

Comment:
NAB: you could DRY all these up by creating a similar `getPolicy()` method and then also ensuring that `getPolicy()` isn't exported in the unit tests.

Reply:
@nkdengineer Also should do the same thing in src/libs/actions/IOU.ts

Reply:
Could you test again to make sure everything works well after these updates?

Reply:
@DylanDylann please check again, i updated code and tested well

Reply:
@nkdengineer Look good but `getPolicy` function is duplicated

@tgolen Should we create a utils function like this
```
function getPolicy(policyID: string | undefined, allPolicies): OnyxTypes.Policy | EmptyObject {
    if (!allPolicies || !policyID) {
        return {};
    }
    return allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] ?? {};
}

```

And export it to use in all files

Reply:
Heh, the balance of DRYness here is difficult to get right. As soon as you export that method, we're back in the same spot that this PR is attempting to clean up. So, I think the DRYness here is acceptable IMO.

Reply:
> So, I think the DRYness here is acceptable IMO.

I realized right after I posted it that this might not be clear. I am suggested that we leave the duplicated methods and that it is OK in this instance for it to not be DRY so that no one is tempted to use the exported method when they shouldn't.